examples/analytics.amp.html

```diff
@@ -55,6 +55,9 @@
       "vars": {
         "eventName": "page-loaded",
         "eventId": "42"
+      },
+      "extraUrlParams": {
```

Comment:
Not really. replaceMap is used when you want to rename the values provided in extraUrlParams. If nothing is specified, the params get added as they are.